pkg/driver/controller.go

```diff
@@ -174,6 +175,12 @@ func (d *controllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
 		case PVNameKey:
 			volumeTags[PVNameTag] = value
 			tProps.PVName = value
+		case BlockExpressKey:
+			if value == "true" {
+				blockExpress = true
+			} else {
+				blockExpress = false
+			}
```

Comment:
np: Don't really need the `else` clause.

Reply:
Fair enough, changed this for `isEncrypted` to be consistent